docs/modules/jenkins-shared-library/partials/odsComponentPipeline.adoc

```diff
@@ -251,6 +251,14 @@ Configuring of a customized slave container in a `Jenkinsfile`:
 odsComponentPipeline(
   projectId: projectId,
   podContainers: [
+    projectId: projectId,
+    componentId: componentId,
+    branchToEnvironmentMapping: [
+      "master": "prod",
+      "develop": "dev",
+      "hotfix/": "hotfix",
+      "*": "review"
+    ],
```

Comment:
This is the wrong level, right? It should be outside `podContainers`. Also note that `projectId` and `componentId` are no longer required. 

To focus on the "customisation" aspect, can we just have an empty `branchToEnvironmentMapping`?

`branchToEnvironmentMapping: [:]` is allowed.